crates/holochain_integrity_types/src/info.rs

```diff
@@ -253,3 +253,9 @@ impl From<u8> for ZomeDependencyIndex {
         Self(v)
     }
 }
+
+impl From<()> for ZomeEntryTypesKey {
+    fn from(_: ()) -> Self {
+        unimplemented!("Should not ever be used")
+    }
+}
```

Comment:
Seems fishy, why is this needed?

Reply:
To satisfy the type bounds on `to_type` when we want to ignore one of the output types